x-pack/platform/plugins/shared/triggers_actions_ui/public/plugin.ts

```diff
@@ -495,9 +509,23 @@ export class Plugin
       getRuleSnoozeModal: (props: RuleSnoozeModalProps) => {
         return getRuleSnoozeModalLazy(props);
       },
+      getUntrackModal: (props: UntrackAlertsModalProps) => {
+        return getUntrackModalLazy(props);
+      },
```

Comment:
```suggestion
      getUntrackModal: UntrackAlertsModal,
```

I don't know why they have this pattern here, perhaps someone from the maintaining team can respond, but this works for me locally. Probably best to just keep it this way as they have others around it that follow the same pattern.